docs/sources/auth/saml.md

```diff
@@ -0,0 +1,136 @@
++++
+title = "SAML Authentication"
+description = "Grafana SAML Authentication"
+keywords = ["grafana", "saml", "documentation", "saml-auth"]
+aliases = ["/auth/saml/"]
+type = "docs"
+[menu.docs]
+name = "SAML"
+parent = "authentication"
+weight = 5
++++
+
+# SAML Authentication
+
+> Only available in Grafana v6.3 and above.
+
+The SAML authentication integration allows your Grafana users to log in by using an external SAML Identity Provider (IdP). To enable this, Grafana becomes a Service Provider (SP) in the authentication flow, interacting with the IdP to exchange user information.
+
+# Supported SAML
+
+The SAML single-sign-on (SSO) standard is varied and flexible. Our implementation contains the subset of features needed to provide a smooth authentication experience into Grafana.
+
+> Should you encounter any problems with our implementation, please don't hesitate to contact us.
+
+At the moment of writing, we support:
+
+1. From the Service Provider (SP) to the Identity Provider (IdP)
+
+  - `HTTP-POST` binding 
+  - `HTTP-Redirect` binding 
+
+2. From the Identity Provider (IdP) to the Service Provider (SP)
+
+  - `HTTP-POST` binding 
+
+# Set up SAML Authentication
+To use the SAML integration, you need to enable SAML in the [main config file]({{< relref "installation/configuration.md" >}}).
+
+```bash
+[auth.saml]
+# Defaults to false. If true, the feature is enabled.
+enabled = true
+
+# Base64-encoded public X.509 certificate. Used to sign requests to the IdP
+certificate =
+
+# Path to the public X.509 certificate. Used to sign requests to the IdP
+certificate_path = 
+
+# Base64-encoded private key. Used to decrypt assertions from the IdP
+private_key =
+
+# Path to the private key. Used to decrypt assertions from the IdP
+private_key_path = 
+
+# Base64-encoded IdP SAML metadata XML. Used to verify and obtain binding locations from the IdP
+idp_metadata =
+
+# Path to the SAML metadata XML. Used to verify and obtain binding locations from the IdP
+idp_metadata_path = 
+
+# Duration, since the IdP issued a response and the SP is allowed to process it. Defaults to 90 seconds.
+max_issue_delay = 
+
+# Duration, for how long the SP's metadata should be valid. Defaults to 48 hours.
+metadata_valid_duration = 
+```
+
+Important to note:
+
+- like any other Grafana configuration, use of [environment variables for these options is supported]({{< relref "installation/configuration/#using-environment-variables" >}})
```

Comment:
Nuh, it's not going to build that way as well, you need to do `[environment variables for these options is supported]({{< relref "installation/configuration.md#using-environment-variables" >}})`.

It's a good idea to run `make docs` to make sure everything is all right

Reply:
Ah-ha! So that's the trick, I noticed we had no docs related tasks within our main `Makefile`. Turns out the docs have their own Makefile. I was blind, but no longer - thanks!